executor/aggregate_test.go

```diff
@@ -710,3 +710,14 @@ func (s *testSuite1) TestIssue10098(c *C) {
 	tk.MustExec("insert into t values('1', '222'), ('12', '22')")
 	tk.MustQuery("select group_concat(distinct a, b) from t").Check(testkit.Rows("1222,1222"))
 }
+
+func (s *testSuite1) TestIssue10608(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
+	tk.MustExec(`drop table if exists t, s;`)
+	tk.MustExec("create table t(a int)")
```

Comment:
this test case currently does not cover the bug since it can't reproduce the bug.

we should modify this line to `create table t(a int primary key);`

Reply:
What do you mean by ` it can't reproduce the bug`?

Reply:
**I expect**
this test case should fail in previous commits(version), such as `3.0.0-beta`
it should return error `1105 - subquery returns more than 1 row`

**what I see instead**
it passed.

-----------------

> What do you mean by it can't reproduce the bug?

this test case can't prove this PR fix the bug described in issues 10608, because this test case will always pass.